README.md

```diff
@@ -6,57 +6,59 @@ Made with love and elixir.
 
 Please note that this project is released with a [Contributor Code of Conduct](code_of_conduct.md). By participating in this project you agree to abide by its terms.
 
-[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md) 
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
 
+# Prerequisites
+
+PostgreSQL >= 10
+
+Elixir >= 1.10
+
+    $ mix deps.get / compile
+    /usr/lib/erlang/lib/parsetools-2.2/include/yeccpre.hrl: no such file or directory
+
+    Install erlang-dev and erlang-parsetools packages.
+
+    $ mix phx.server
+    [error] calling logger:remove_handler(default) failed: :error {:badmatch, {:error, {:not_found, :default}}}
+    ** (Mix) Could not start application xmerl: could not find application file: xmerl.app
+
+    Install erlang-xmerl
+
+RabbitMQ (version?)
+
+NodeJS (>= 10?) / npm >= 7?
 
 # Development setup
 
 Proca backend is an Elixir app that uses PostgreSQL as data store and RabbitMQ
-for data processing. The PostgreSQL is managed by migrations. The RabbitMQ is
-(re)configured live when the app is run, it only needs predentials to access it.
+for data processing.
 
-1. Databases: create
+The script utils/configure-development-environment.sh will setup PostgreSQL, the Erlang / Elixir / Pheonix server, the RabbitMQ server and runs npm install in the assets directory.
 
-```
-sudo -i
-su postgres
-createuser -P proca
-# input "proca" as password
-createdb -O proca proca
-createdb -O proca proca_test
-exit
-```
+You'll need sudo to run the parts of the script that configure PostgreSQL.
 
-3. Databases: initialize
-```
-mix ecto.migrate
-mix run priv/repo/seeds.exs
-# same for test db
-env MIX_ENV=test mix ecto.migrate
-env MIX_ENV=test  mix run priv/repo/seeds.exs
-```
+$ ./utils/configure-development-environment.sh
 
-3. Create RabbitMQ
+If things go wrong, you can refer to the script for the details of installing and configuring each piece.
 
-```
-./utils/create-queue
-sudo -i
-echo "172.19.0.3  rabbitmq.docker" >> /etc/hosts
-# add proca user and proca vhost
-./utils/create-queue-creds
-```
+Once the installation / configuration is complete you can setup an Organization and Admin user:
 
-Login to http://rabbitmq.docker:15672/ with guest/guest and add:
-- virtual host called "proca" (not "/proca")
-- user called proca (pass proca) with full access to virtual host "proca"
+    # Set up your ENV
+    $ export ADMIN_EMAIL=you@example.com MIX_ENV=dev ORGANIZATION='Fix the Status Quo'
```

Comment:
Org name just not have spaces. It can have alphanum, hyphen, dot. A long name is called a title everywhere to avoid confusion.